### DISCUSSION TITLE
Is there is a way to combine several *.pwmx files to one?

### DISCUSSION THREAD

**Discussion by Manofjesus** [sn4k3/UVtools#617] [Q&A]
Thank you for the gorgeous software !!!
For the purposes of finding best AA settings for the resin with Mono X i think of idea to generate some *.pwmx files with different AA settings and then to combine them in one printing file. The area of the single AA model will not too big, let it be 30x30 mm. Then for example i have 8 different files with different AA settings. How them combine into one printing file and distribute on the printing field like 4 x 2 (two taws with 4 samples in each)?

**Comment by sn4k3** [comment]
You can combine files but why? Anycubic use fake AA don't worth anyway. If you want to use AA go with x4 or x8

**Reply by Manofjesus** [reply]
It is look like new hardware releases has a better AA option. My mono X goes with 3.6.4 version. So i was thinking of fine tuning AA settings to clarify all this thougths. Also you answered that it can be combined - how this can be done, by some scripting?

**Reply by sn4k3** [reply]
> It is look like new hardware releases has a better AA option. 

You mean firmware? They can fix some issues like artifacts and unable to proper print with AA but they will never change how AA work on present Anycubic machines, being fake AA you have little benefit, more trouble than benefit imo.

To combine files use Layer actions -> Import layers and use MergeMax option

**Reply by Manofjesus** [reply]
I understand your concern, than without hardware support firmware can just emulate some sort of AA likeness. But i mean not just new firmware alone. This firmware can't be installed on older machines due to another hardware revision, that is the point.
